legal-api/src/legal_api/models/review_result.py

```diff
@@ -69,3 +70,14 @@ def get_last_review_result(cls, review_id) -> ReviewResult:
                              order_by(ReviewResult.creation_date.desc()).
                              first())
         return review_result
+
+    @property
+    def json(self) -> dict:
+        """Return ReviewResult as a JSON object."""
+        return {
+            'status': self.status.name,
+            'comments': self.comments,
+            'reviewer': (User.find_by_id(self.review_id)).display_name,
```

Comment:
Double check if this is correct as well for reviewer

Reply:
```suggestion
            'reviewer': self.reviewer.display_name,
```
can you check this?

Reply:
did you check this?